src/werkzeug/routing/rules.py

```diff
@@ -815,13 +826,13 @@ def _join(parts: list[ast.AST]) -> ast.AST:
         # bad line numbers cause an assert to fail in debug builds
         for node in ast.walk(module):
             if "lineno" in node._attributes:
-                node.lineno = 1
+                node.lineno = 1  # type: ignore[attr-defined]
```

Comment:
These type ignores show up as unused right now with the current version of typeshed; I'll just remove them and they can be added when you pull in the new version of mypy.